tables/de-g0-bd.utb

```diff
@@ -0,0 +1,91 @@
+# liblouis: German Grade 0 Braille (bidirectional
+#
+# Das System der deutschen Blindenschrift (2015/2018)
+# http://www.bskdl.org/textschrift.html
+#
+# Bemerkung:
+# - Gross-/Kleinschreibung wird nicht berücksichtigt (d.h. für
+#   Abkürzungen u.ä. braucht es Vorprogramm)
+# - Akzente: Hinweis auf die beiden Tabellen (bis jetzt nur als
+#   Kommentar in de-g0.utb)
```

Comment:
This text needs to be corrected.

Reply:
> Hmm, yes, I did consider it, but what does it actually mean, and what should we say instead, or should I just remove it?

Christian will do it.

Reply:
@egli

Reply:
> Gross-/Kleinschreibung wird nicht berücksichtigt (d.h. für Abkürzungen u.ä. braucht es Vorprogramm)

Capitalization is ignored, i.e. for abbrevs you will need a pre-processor

> Akzente: Hinweis auf die beiden Tabellen (bis jetzt nur als Kommentar in de-g0.utb)

Not sure about this one. I think there is some special treatment of accents and maybe this is only documented in `de-g0.utb`?

Reply:
@egli You marked this resolved but it's not fixed.

Reply:
I think I fixed this in 7259d41c4986785f599b653be2a3f2b309f77340

Reply:
OK I guess it was not pushed yet.